test/fake-payload.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
this should be in a folder called fixtures and you should have one for each state that you are testing for